src/backend/web/validation.js

```diff
@@ -97,6 +97,10 @@ const queryValidationRules = [
     .withMessage('invalid filter value')
     .bail(),
 ];
+// id param for delete post
```

Comment:
In this case, it's safe to reuse the `postsIdParamValidationRules`, since it's the same value (i.e., an `id` for a post).